src/v3/src/transformer/selectAuthenticator/transformSelectOVMethodVerify.ts

```diff
@@ -76,7 +77,11 @@ export const transformSelectOVMethodVerify: IdxStepTransformer = ({ transaction,
       'authenticator.methodType',
       uischema.elements as UISchemaElement[],
     );
-    uischema.elements = uischema.elements.concat(buttonElements);
+    const authenticatorListElement: AuthenticatorButtonListElement = {
+      type: 'AuthenticatorButtonList',
+      options: { buttons: buttonElements },
+    };
+    uischema.elements = uischema.elements.concat([authenticatorListElement]);
```

Comment:
Is there any reason we want to use concat vs push? Only asking for the sake of consistency

Reply:
Oh yea, concat is not necessary here, just pushing the element is more than enough. Updating.

Reply:
Ok sounds good! There's a few other instances in other transformers as well